src/cr/cube/cubepart.py

```diff
@@ -337,17 +348,15 @@ def inserted_row_idxs(self):
     @lazyproperty
     def insertions(self):
         """Returns masked array with residuals for insertions
-
-                 0     1	 2	     3	    4	    5	    6
-           0   inf   inf   inf	   inf	  inf	 -2.9	  inf
-           1   inf	 inf   inf	   inf	  inf	 -4.3	  inf
-           2   2.5	 1.3   3.3	 -0.70	-7.25	 -6.52	 2.25
-           3   inf	 inf   inf	   inf	  inf	 -2.51	  inf
-           4  -1.16	 2.20  5.84	  1.78	-8.48	 -5.92	 0.93
-           5   inf   inf   inf	   inf	  inf	  9.70	  inf
-
-           Only the insertions residuals are showed in a inf masked array
-        """
+              0     1     2     3      4       5       6
+        0   inf   inf   inf   inf    inf    -2.9     inf
+        1   inf   inf   inf   inf    inf    -4.3     inf
+        2   2.5   1.3   3.3  -0.70  -7.25   -6.52    2.25
+        3   inf   inf   inf   inf    inf    -2.51    inf
+        4  -1.16  2.20  5.84  1.78  -8.48   -5.92    0.93
+        5   inf   inf   inf   inf    inf     9.70    inf
+
+        Only the insertions residuals are showed in a inf masked array"""
```

Comment:
👍  this scattered alignment with tabs in it has been bugging me for months :)

Final `"""` should appear on its own line. From the PEP257 link above:

> Unless the entire docstring fits on a line, place the closing quotes on a line by themselves.